docs/configuration.md

```diff
@@ -1292,6 +1292,16 @@ Apart from these, the following properties are also available, and may be useful
     Controls whether to clean checkpoint files if the reference is out of scope.
   </td>
 </tr>
+<tr>
+  <td><code>spark.network.io.preferDirectBufs</code></td>
```

Comment:
@vanzin do you want to document these? I don't see that the others are. I've thought of them as like the many other unadvertised settings we have for things like this

Reply:
I think this one is fine to document. The shuffle option is documented, and this one is useful for similar reasons.

Reply:
OK that's fine. But I think this belongs in the networking rather than memory section. Maybe sort it after spark.network.timeout